test/Tests/src/Data/Text/Regex_Spec.enso

```diff
@@ -25,4 +25,6 @@ spec =
             pattern = "http://example.com"
             Regex.escape pattern . should_equal "\Qhttp://example.com\E"
 
+    ## TODO: Missing tests for No_Such_Group_Error
```

Comment:
This is more of a wishlist since `No_Such_Group_Error.to_display_text` pattern matches on `Text` and `Integer` types but I only discovered those via a simple grep because all tests were passing.